url.bs

```diff
@@ -1325,12 +1331,11 @@ different document encoding. Using the <a>UTF-8</a> encoding everywhere solves t
  <li><p>If <var>url</var>'s <a for=url>scheme</a> is not
  "<code>blob</code>", return <var>url</var>.
 
- <li><p>If <var>url</var>'s <a for=url>path</a> <a for=list>is empty</a> or <var>url</var>'s
- <a for=url>path</a>[0] is not in the <a>Blob URL Store</a>, then return <var>url</var>.
- [[!FILEAPI]]
+ <li><p>Set <var>url</var>'s <a for=url>cached origin</a> to the result of
+ <a for="blob URL" lt="resolve the origin">resolving the origin of the blob URL</a> <var>url</var>.
 
- <li><p>Set <var>url</var>'s <a for=url>object</a> to the entry in the <a>Blob URL Store</a>
- corresponding to <var>url</var>'s <a for=url>path</a>[0].
+ <li><p>Set <var>url</var>'s <a for=url>object</a> to the result of
+ <a for="blob URL" lt="resolve">resolving the blob URL</a> <var>url</var>.
```

Comment:
The "[resolving the blob URL](https://w3c.github.io/FileAPI/#blob-url-resolve)" algorithm can return *failure*. I guess the url’s `object` should be set to `null` then.

Reply:
Done